notebooks/240-named-entity-recognition/240-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>it is not data preprocessing task. can we combain first and second sentence in one or remove first sentence at all (just Named Entity Recognition is a natural language processing method ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done